fetch.bs

```diff
@@ -6633,7 +6653,8 @@ method steps are:
    <li><p>If <var>response</var> is a <a>network error</a>, then <a for=/>reject</a> <var>p</var>
    with a {{TypeError}} and terminate these substeps.
 
-   <li><p>Associate <var>responseObject</var> with <var>response</var>.
+   <li><p>Set <var>responseObject</var> to the result of <a for=Response>creating</a> a {{Response}}
```

Comment:
If you're worried about races between this and the "Abort fetch" line above, you could check the state `locallyAborted` and abort these steps if it's true.

Reply:
That's step 1 here so I think we're good, no?